File: Revisão_Aritmeticos/operadores_aritmeticos.java
package Revisão_Aritmeticos;

public class operadores_aritmeticos {
    public static void main(String[] args) {
        
        int a = 15;
        int b = 4;
        System.out.println("a + b = " + (a + b));
        System.out.println("a - b = " + (a - b));
        System.out.println("a * b = " + (a * b));
        System.out.println("a / b = " + ((double)a / b));
        System.out.println("a / b (int) = " + (a / b));
        System.out.println("a % b = " + (a % b));
        System.out.println("a ^ b (pow) = " + (int)Math.pow(a, b));
    }
}

File: Revisão_Aritmeticos/operadores_aritmeticos.c
#include <stdio.h>
#include <math.h>
int main() {
    int a = 15, b = 4;
    printf("a + b = %d\n", a + b);
    printf("a - b = %d\n", a - b);
    printf("a * b = %d\n", a * b);
    printf("a / b = %.2f\n", (double)a / b);
    printf("a / b (int) = %d\n", a / b);
    printf("a %% b = %d\n", a % b);
    printf("a ^ b (pow) = %.0f\n", pow(a, b));
    return 0;
}